spec/services/support_interface/candidates/bulk_unsubscribe_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::Candidates::BulkUnsubscribe do
+  describe '.bulk_unsubscribe' do
```

Comment:
Could we add a test that shows if an email address is included that doesn't actually belong to a candidate, it doesn't break the job? 

I'm thinking of the scenario where a candidate changes their email address between clicking the unsubscribe button via notify and someone running this job.